### DISCUSSION TITLE
When I use a new-line the cursor is displaced by at least 10 spaces

### DISCUSSION THREAD

**Discussion by francisco-lafe** [JanDeDobbeleer/oh-my-posh#2379] [Troubleshoot] [answered]
Hello.

This is a weird error and I don't know if it's OMP or my custom theme (attached). [mytheme.omp.json.zip](https://github.com/JanDeDobbeleer/oh-my-posh/files/8835922/mytheme.omp.json.zip)
As you can see, on the same line I have the "cursor" on the beginning of the line, but if I do  SHIFT+ENTER (Newline) the cursor gets displaced by a lot: 
![2022-06-03 - 1020  - powershell](https://user-images.githubusercontent.com/90066435/171959620-b637a670-64ff-41c9-81c2-d58cc62a8c18.gif)
What is going on? 
Windows Powershell:
![image](https://user-images.githubusercontent.com/90066435/171963120-14d24745-2652-421a-8acf-1138ef158711.png)

Thanks.

**Comment by JanDeDobbeleer** [comment]
Which shell is this?

**Reply by francisco-lafe** [reply]
Sorry, I forgot to specify. 
I'm using Windows Powershell 5.1 
![image](https://user-images.githubusercontent.com/90066435/171963104-baa1e610-bc23-4f15-b239-720ef37c9191.png)

**Comment by francisco-lafe** [comment, marked as answer]
Refer to: 
[#2395](https://github.com/JanDeDobbeleer/oh-my-posh/issues/2395#issuecomment-1148936192) for solution
